indico/modules/events/sessions/operations.py

```diff
@@ -36,6 +37,17 @@ def create_session(event, data):
     return event_session
 
 
+def create_session_block(session_, data):
+    block = SessionBlock(session=session_)
+    block.populate_from_dict(data)
+    db.session.flush()
+    session_.event_new.log(EventLogRealm.management, EventLogKind.positive, 'Sessions',
+                           "Session block \"{}\" for session \"{}\" has been created"
```

Comment:
do we really want to start escape orgies instead of using single quotes (yes, even if it's an string containing natural language - practicality beats purity)